gramps_webapi/api/__init__.py

```diff
@@ -86,10 +86,16 @@ def register_endpt(resource: Type[Resource], url: str, name: str):
 register_endpt(TranslationsResource, "/translations/", "translations")
 # Relation
 register_endpt(
-    RelationResource, "/relations/<string:handle1>/<string:handle2>", "relations",
+    RelationResource,
+    "/relations/<string:handle1>/<string:handle2>",
+    "relations",
 )
 # Metadata
-register_endpt(MetadataResource, "/metadata/<string:datatype>", "metadata")
+register_endpt(MetadataResource, "/metadata/", "metadata")
```

Comment:
Why was this changed here, given it is unrelated to reports?

Reply:
I probably mixed something up, it'll get fixed by time this is ready for merging.  I have some code laying around that refactored the metadata endpoint.